metals/src/main/scala/scala/meta/internal/metals/FormattingProvider.scala

```diff
@@ -271,6 +271,7 @@ final class FormattingProvider(
       downloadingScalafmt = Promise()
       statusBar.trackSlowFuture(
         "Loading Scalafmt",
+        quietLogs = true,
```

Comment:
No interesting logs will be shown here. And even in case of an error, the log will be hidden afterwards. This operation is normally too quick to make sense to open the logs.